build/storage/tests/it/ptf_tests/tests/test_hot_plug.py

```diff
@@ -0,0 +1,166 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+from ptf.base_tests import BaseTest
+from system_tools.config import TestsConfig
+from system_tools.errors import CommandException
+from system_tools.test_platform import KVMPlatformsFactory
+
+
+class TestMinHotPlug(BaseTest):
+    def setUp(self):
+        self.platforms_factory = KVMPlatformsFactory()
```

Comment:
You probably missed my comment
https://github.com/intelfisz/ipdk/pull/10#discussion_r1036086189

Reply:
changed